perun-core/src/test/java/cz/metacentrum/perun/core/entry/ConsentsManagerEntryIntegrationTest.java

```diff
@@ -468,6 +418,259 @@ public void changeConsentStatusToSameValue() throws Exception {
 		assertThatExceptionOfType(InvalidConsentStatusException.class).isThrownBy(() -> perun.getConsentsManagerBl().changeConsentStatus(sess, consent, status));
 	}
 
+	@Test
+	public void evaluateConsentsIsTurnedOffOnInstance() throws Exception {
+		System.out.println(CLASS_NAME + "evaluateConsentsIsTurnedOffOnInstance");
+
+		Service service = setUpService();
+		Vo vo = setUpVo();
+		Member member = perun.getMembersManager().createMember(sess, vo, user);
+		Resource resource = setUpResource(facility, vo);
+		perun.getResourcesManagerBl().assignService(sess, resource, service);
+		AttributeDefinition attrDef = setUpUserAttributeDefinition("testUserAttribute");
+		perun.getServicesManagerBl().addRequiredAttribute(sess, service, attrDef);
+
+		List<Member> allowedMembers = perun.getConsentsManagerBl().evaluateConsents(sess, service, facility, List.of(member));
+		// didn't sign any consent but should be still returned because consents are turned off in the instance config
+		assertThat(allowedMembers).containsExactly(member);
+	}
+
+	@Test
+	public void evaluateConsentsIsTurnedOffOnHub() throws Exception {
+		System.out.println(CLASS_NAME + "evaluateConsentsIsTurnedOffOnHub");
+
+		Service service = setUpService();
+		Vo vo = setUpVo();
+		Member member = perun.getMembersManager().createMember(sess, vo, user);
+		Resource resource = setUpResource(facility, vo);
+		perun.getResourcesManagerBl().assignService(sess, resource, service);
+		AttributeDefinition attrDef = setUpUserAttributeDefinition("testUserAttribute");
+		perun.getServicesManagerBl().addRequiredAttribute(sess, service, attrDef);
+
+		ConsentHub consentHub = consentsManagerEntry.getConsentHubByFacility(sess, facility.getId());
+		consentHub.setEnforceConsents(false);
+		consentsManagerEntry.updateConsentHub(sess, consentHub);
+
+		List<Member> allowedMembers;
+		try {
+			BeansUtils.getCoreConfig().setForceConsents(true);
+			allowedMembers = perun.getConsentsManagerBl().evaluateConsents(sess, service, facility, List.of(member));
+		} finally {
+			BeansUtils.getCoreConfig().setForceConsents(false);
```

Comment:
just a thought - maybe we should set it to previous value, not necessarily `false`? it could be retrieved once for the whole class and then used by different tests.